mmaction/models/roi_heads/shared_heads/lfb.py

```diff
@@ -86,15 +86,15 @@ def __init__(self,
         self.dataset_modes = dataset_modes
         self.device = device
 
-        if 'LOCAL_RANK' in os.environ:
-            local_rank = int(os.environ['LOCAL_RANK'])
-        else:
-            rank, world_size = get_dist_info()
-            gpus_per_node = torch.cuda.device_count()
-            local_rank = rank % gpus_per_node
-
+        rank, world_size = get_dist_info()
         # Loading LFB
         if self.device == 'gpu':
+            if 'LOCAL_RANK' in os.environ:
+                local_rank = int(os.environ['LOCAL_RANK'])
+            else:
+                gpus_per_node = torch.cuda.device_count()
```

Comment:
What is the motivation of moving this part

Reply:
In previous version, on cpu only device, `gpus_per_node` is equal to 0, and `local_rank = rank % gpus_per_node` will cause `ZeroDivisionError`.